From 38e3e5ad36ac3dda51b9b40eb1651e311187e76d Mon Sep 17 00:00:00 2001
From: Lawrence Mok <lawrence.mok@amlogic.com>
Date: Mon, 16 Dec 2013 14:09:36 -0800
Subject: [PATCH 2532/5965] m8 defconfig: make savedefconfig

No difference in resulting .config.  Use 'make savedefconfig' and
check-in resulting defconfig as meson8_defconfig.
---
 arch/arm/configs/meson8_defconfig | 191 ++++++++++--------------------
 1 file changed, 65 insertions(+), 126 deletions(-)

diff --git a/arch/arm/configs/meson8_defconfig b/arch/arm/configs/meson8_defconfig
index 9bc5b8de07ed..6251e1785c9b 100755
--- a/arch/arm/configs/meson8_defconfig
+++ b/arch/arm/configs/meson8_defconfig
@@ -4,7 +4,6 @@ CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
 CONFIG_BSD_PROCESS_ACCT=y
 CONFIG_IKCONFIG=y
-CONFIG_IKCONFIG_PROC=n
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_RD_BZIP2=y
 CONFIG_RD_LZMA=y
@@ -19,16 +18,21 @@ CONFIG_MESON_LEGACY_REGISTER_API=y
 CONFIG_ARM_ERRATA_751472=y
 CONFIG_ARM_ERRATA_754322=y
 CONFIG_ARM_ERRATA_764369=y
-CONFIG_ARCH_MESON8=y
 CONFIG_SMP=y
 CONFIG_PREEMPT=y
 CONFIG_AEABI=y
 CONFIG_HIGHMEM=y
 # CONFIG_COMPACTION is not set
+CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_GOV_ONDEMAND=y
+CONFIG_CPU_FREQ_GOV_INTERACTIVE=y
+CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
+CONFIG_AMLOGIC_MESON_CPUFREQ=y
 CONFIG_VFP=y
 CONFIG_NEON=y
 CONFIG_BINFMT_MISC=y
-CONFIG_NET=y
+CONFIG_WAKELOCK=y
+CONFIG_PM_RUNTIME=y
 CONFIG_PACKET=y
 CONFIG_PACKET_DIAG=y
 CONFIG_UNIX=y
@@ -135,18 +139,18 @@ CONFIG_NET_CLS_ACT=y
 CONFIG_NET_ACT_POLICE=y
 CONFIG_NET_ACT_GACT=y
 CONFIG_NET_ACT_MIRRED=y
-
-CONFIG_WIRELESS=y
+CONFIG_BT=y
+CONFIG_BT_RFCOMM=y
+CONFIG_BT_RFCOMM_TTY=y
+CONFIG_BT_BNEP=y
+CONFIG_BT_BNEP_MC_FILTER=y
+CONFIG_BT_BNEP_PROTO_FILTER=y
+CONFIG_BT_HIDP=y
+CONFIG_BT_HCIUART=y
+CONFIG_BT_HCIUART_H4=y
 CONFIG_CFG80211=y
-CONFIG_CFG80211_DEFAULT_PS=y
 CONFIG_MAC80211=y
-CONFIG_MAC80211_HAS_RC=y
-CONFIG_MAC80211_RC_MINSTREL=y
-CONFIG_MAC80211_RC_MINSTREL_HT=y
-CONFIG_MAC80211_RC_DEFAULT_MINSTREL=y
-CONFIG_MAC80211_RC_DEFAULT="minstrel_ht"
-CONFIG_WAKELOCK=y
-CONFIG_SUSPEND=y
+CONFIG_RFKILL=y
 CONFIG_I2C_AML=y
 CONFIG_SARADC_AM=y
 # CONFIG_MESON_INPUT_REMOTE is not set
@@ -159,32 +163,50 @@ CONFIG_GOODIX_GT81XX_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_GSLX680_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_GOODIX_GT9XX_CAPACITIVE_TOUCHSCREEN=y
 CONFIG_TOUCHSCREEN_CT36X=y
+CONFIG_SENSOR_DEVICES=y
+CONFIG_GRAVITY_BMA250=y
+CONFIG_GRAVITY_BMA222=y
+CONFIG_GRAVITY_MMA7660=y
+CONFIG_GRAVITY_LIS3DH=y
+CONFIG_GRAVITY_MMA8452=y
+CONFIG_GRAVITY_MC32X0=y
+CONFIG_GRAVITY_DMARD06=y
+CONFIG_GRAVITY_MMA865X=y
+CONFIG_GRAVITY_LSM303D=y
+CONFIG_GRAVITY_DMARD10=y
+CONFIG_GRAVITY_KXTJ9=y
+CONFIG_GRAVITY_STK8312=y
+CONFIG_GRAVITY_MM3A310=y
+CONFIG_GRAVITY_MXC622X=y
+CONFIG_LIGHT_CM36283=y
+CONFIG_LIGHT_CM3232=y
+CONFIG_LIGHT_LTR501=y
+CONFIG_LIGHT_EPL6814=y
+CONFIG_LIGHT_ISL29023=y
+CONFIG_LIGHT_STK220X=y
+CONFIG_LIGHT_LTR558=y
 CONFIG_AML_GPIO_KEY=y
-CONFIG_AML_PMU_ALGORITHM_SUPPORT=y
 CONFIG_RICOH_PMU=y
-CONFIG_RN5T618=y
+CONFIG_SECURITYKEY=y
 CONFIG_EFUSE=y
 CONFIG_AMLOGIC_USB=y
 CONFIG_MMC_AML=y
-# CONFIG_MMC_AML_DEBUG is not set
 CONFIG_AMLOGIC_SPI_NOR=y
 CONFIG_AML_NEXT_GEN_NAND=y
+CONFIG_AML_NAND_KEY=y
 CONFIG_AM_ETHERNET=y
 CONFIG_AM_WIFI=y
 CONFIG_AM_WIFI_SD_MMC=y
-CONFIG_BCMDHD_POWER_OFF_IN_SUSPEND=y
-CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=m
-CONFIG_BCMDHD_OOB=y
 CONFIG_AM_WIFI_USB=y
+CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=m
+CONFIG_BCMDHD_POWER_OFF_IN_SUSPEND=y
 CONFIG_RTL8188EU=m
 CONFIG_RTL8188EU_MP=m
-CONFIG_AM_ETHERNET_DEBUG_LEVEL=1
+CONFIG_BT_DEVICE=y
 CONFIG_VSYNC_RDMA=y
 CONFIG_AM_VIDEO=y
 CONFIG_AM_ENCODER=y
 CONFIG_AM_TV_OUTPUT2=y
-CONFIG_V4L_AMLOGIC_VIDEO=y
-CONFIG_AM_HDMI_ONLY=y
 CONFIG_FB_AM=y
 CONFIG_FB_OSD2_CURSOR=y
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
@@ -193,21 +215,34 @@ CONFIG_FB_OSD2_DEFAULT_WIDTH_VIRTUAL=32
 CONFIG_FB_OSD2_DEFAULT_HEIGHT_VIRTUAL=32
 CONFIG_AM_FB_EXT=y
 CONFIG_AMLOGIC_BACKLIGHT=y
-CONFIG_AM_GE2D=y
 CONFIG_AM_LOGO=y
-CONFIG_TVIN=y
-CONFIG_TVIN_VDIN=y
+CONFIG_AM_HDMI_ONLY=y
 CONFIG_VIUIN=y
 CONFIG_TVIN_ISP=y
 CONFIG_POST_PROCESS_MANAGER=y
 CONFIG_POST_PROCESS_MANAGER_PPSCALER=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE=y
+CONFIG_AMLOGIC_CAPTURE_FRAME_ROTATE=y
+CONFIG_AMLOGIC_VM_DISABLE_VIDEOLAYER=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_GT2005=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0308=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0328=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_GC2035=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_OV5640=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_SP0A19=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_HI253=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_HM2057=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_OV5647=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_AR0543=y
+CONFIG_VIDEO_AMLOGIC_CAPTURE_AR0833=y
 CONFIG_V4L_AMLOGIC_VIDEO=y
 CONFIG_V4L_AMLOGIC_VIDEO2=y
+CONFIG_AMLOGIC_IONVIDEO=y
 CONFIG_DEINTERLACE=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
-CONFIG_MTD=y
-CONFIG_MTD_BLOCK=y
+CONFIG_CMA=y
+CONFIG_CMA_SIZE_MBYTES=12
 CONFIG_PROC_DEVICETREE=y
 CONFIG_OF_SELFTEST=y
 CONFIG_BLK_DEV_LOOP=y
@@ -217,7 +252,6 @@ CONFIG_BLK_DEV_SR=y
 CONFIG_SCSI_MULTI_LUN=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
-CONFIG_PHYLIB=y
 CONFIG_PPP=y
 CONFIG_PPP_BSDCOMP=y
 CONFIG_PPP_DEFLATE=y
@@ -234,24 +268,16 @@ CONFIG_INPUT_EVDEV=y
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_UINPUT=y
 CONFIG_DEVPTS_MULTIPLE_INSTANCES=y
-CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
-CONFIG_SPI=y
-CONFIG_MEDIA_SUPPORT=y
-CONFIG_SOUND=y
-CONFIG_SND=y
-CONFIG_SND_SOC=y
-CONFIG_SND_AML_SOC=y
+CONFIG_SND_USB_AUDIO=y
 CONFIG_SND_AML_M_DUMMY_CODEC=y
 CONFIG_SND_AML_M8=y
-CONFIG_SND_AML_M8_RT5616=y
 CONFIG_SND_AML_M8_PCM=y
+CONFIG_SND_AML_M8_RT5616=y
 CONFIG_UHID=y
-CONFIG_SND_USB=y
-CONFIG_SND_USB_AUDIO=y
 CONFIG_USB=y
 CONFIG_USB_ACM=y
 CONFIG_USB_STORAGE=y
@@ -294,100 +320,13 @@ CONFIG_CRAMFS=y
 CONFIG_CIFS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
-CONFIG_PRINTK_TIME=y
 CONFIG_NLS_UTF8=y
+CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_READABLE_ASM=y
 CONFIG_DEBUG_SECTION_MISMATCH=y
 CONFIG_SCHEDSTATS=y
 CONFIG_TIMER_STATS=y
 CONFIG_DEBUG_INFO=y
-CONFIG_CRYPTO_AUTHENC=y
-CONFIG_CRYPTO_CBC=y
-CONFIG_CRYPTO_ECB=y
-CONFIG_CRYPTO_HMAC=y
-CONFIG_CRYPTO_MD5=y
-CONFIG_CRYPTO_SHA1=y
-CONFIG_CRYPTO_SHA256=y
-CONFIG_CRYPTO_AES=y
-CONFIG_CRYPTO_ARC4=y
-CONFIG_CRYPTO_DES=y
-CONFIG_CRYPTO_DEFLATE=y
-CONFIG_CRC_CCITT=y
-CONFIG_CRC16=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
-CONFIG_LIBCRC32C=y
-CONFIG_AVERAGE=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE=y
-CONFIG_AMLOGIC_CAPTURE_FRAME_ROTATE=y
-CONFIG_AMLOGIC_VM_DISABLE_VIDEOLAYER=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0308=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_GC0328=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_GC2035=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_GT2005=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_OV5640=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_OV5647=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_HI253=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_HM2057=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_AR0543=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_AR0833=y
-CONFIG_VIDEO_AMLOGIC_CAPTURE_SP0A19=y
-CONFIG_TVIN_ISP=y
-CONFIG_TVIN_CSI=y
-CONFIG_SENSOR_DEVICES=y
-CONFIG_GRAVITY_BMA250=y
-CONFIG_GRAVITY_BMA222=y
-CONFIG_GRAVITY_MMA7660=y
-CONFIG_GRAVITY_LIS3DH=y
-CONFIG_GRAVITY_MMA8452=y
-CONFIG_GRAVITY_MC32X0=y
-CONFIG_GRAVITY_DMARD06=y
-CONFIG_GRAVITY_MMA865X=y
-CONFIG_GRAVITY_STK8312=y
-CONFIG_GRAVITY_LSM303D=y
-CONFIG_GRAVITY_DMARD10=y
-CONFIG_GRAVITY_KXTJ9=y
-CONFIG_GRAVITY_MM3A310=y
-CONFIG_GRAVITY_MXC622X=y
-CONFIG_LIGHT_CM36283=y
-CONFIG_LIGHT_CM3232=y
-CONFIG_LIGHT_LTR501=y
-CONFIG_LIGHT_EPL6814=y
-CONFIG_LIGHT_STK220X=y
-CONFIG_LIGHT_LTR558=y
-CONFIG_LIGHT_ISL29023=y
-CONFIG_CMA=y
-CONFIG_CMA_SIZE_MBYTES=12
-CONFIG_CMA_SIZE_SEL_MBYTES=y
-CONFIG_CMA_ALIGNMENT=8
-CONFIG_CMA_AREAS=7
-CONFIG_SECURITYKEY=y
-CONFIG_AML_NAND_KEY=y
-CONFIG_NAND_KEY_BLOCK_NUM=8
-CONFIG_AML_EMMC_KEY=y
-CONFIG_BT=y
-CONFIG_BT_L2CAP=y
-CONFIG_BT_SCO=y
-CONFIG_BT_RFCOMM=y
-CONFIG_BT_RFCOMM_TTY=y
-CONFIG_BT_BNEP=y
-CONFIG_BT_BNEP_MC_FILTER=y
-CONFIG_BT_BNEP_PROTO_FILTER=y
-CONFIG_BT_HIDP=y
-CONFIG_BT_HCIUART=y
-CONFIG_BT_HCIUART_H4=y
-CONFIG_RFKILL=y
-CONFIG_BT_DEVICE=y
-CONFIG_CPU_FREQ=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
-CONFIG_AMLOGIC_MESON_CPUFREQ=y
-CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
-CONFIG_CPU_FREQ_GOV_ONDEMAND=y
-CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
-CONFIG_CPU_FREQ_GOV_INTERACTIVE=y
-CONFIG_AML_DVFS=y
-CONFIG_RTC_CLASS=y
-CONFIG_AML_RTC=y
-CONFIG_PM_RUNTIME=y
-CONFIG_AMLOGIC_IONVIDEO=y
-- 
2.19.0

